pkgs/top-level/python-packages.nix

```diff
@@ -10690,6 +10690,11 @@ self: super: with self; {
     debugger = pkgs.gdb;
   };
 
+  pwntools-full = callPackage ../development/python-modules/pwntools {
```

Comment:
We use [pypi names](https://github.com/NixOS/nixpkgs/issues/245383) in pythonPackages and there is no such package on pypi: https://pypi.org/project/pwntools-full/.

Reply:
Another problem with this is collisions.
You cannot co-install pwntools and pwntools full, because they contain the same files.

Reply:
Sounds like we should just enable everything by default then and let the user trim their pwntools installation if absolutely necessary? As someone who's used pwntools occasionally for CTF stuff, that seems much more convenient for my usage.